implementations/CredenceID.json

```diff
@@ -3,6 +3,7 @@
 	"implementation": "CredenceID",
 	"verifiers": [
 		{
+			"id": "https://verifier.credenceid.com",
```

Comment:
```suggestion
			"id": "https://verifier.credenceid.com/credentials/verify",
```

Reply:
The Digital Bazaar implementation JSON (and possibly others) take this approach.